src/main/resources/application.yml

```diff
@@ -1,6 +1,11 @@
 spring:
   redis:
-    host: ${redis.host}
+    host: your-hostname
+    password: "your-password"
```

Comment:
...and a default password that's empty would also work well for local development.